### DISCUSSION TITLE
Displaying full decimal amounts

### DISCUSSION THREAD

**Discussion by Dilosen** [ghostfolio/ghostfolio#3689] [Q&A]
Is it possible to express the full amount instead of two decimal points? Especially for things like crypto currency 

i.e. 0.0010000 BTC should be visable, but shows 00.00 on the frontend unless clicked,  and even then it is still two decimal points.

**Comment by Dilosen** [comment]
bumping this for a solution.

I need more than 4 decimal points at minimum but only have two and it rounds up. This is not accurate for the activity allotment.

Please help with this

**Comment by DireMunchkin** [comment]
I'd also like a more precise displayed number - With Bitcoin nearly worth $100 k, rounding to 0.01 could have a rounding error of $500 worth in the worst case. That's very imprecise.

**Reply by dtslvr** [reply]
As mentioned [here](https://github.com/ghostfolio/ghostfolio/issues/4085#issuecomment-2509134009), the calculations are done with full precision. Values are rounded for presentation only.

**Comment by maximecurioni** [comment]
@dtslvr thanks for clarifying that the calculations are done with full precision. 

I was wondering if the presentation precision could be a configurable setting for users. I'd like to use a precision of 4 decimal digits for my use-case, but I am sure other users would prefer other values.

As a new user of Ghostfolio, it was a pretty confusing experience to have the quantity of shares rounded up to 2 decimal digits. I had to find this thread and double-check my database rows to make sure that the data was intact 😄

**Comment by reedhaffner** [comment]
Also requesting a feature to see higher decimal precision on the UI. Maybe a hover over the rounded amount could show a tooltip with the accurate balance? This would make it a lot easier to diagnose balances which are slightly off compared to the brokerage.

**Comment by satonak21** [comment]
> Is it possible to express the full amount instead of two decimal points? Especially for things like crypto currency
> 
> i.e. 0.0010000 BTC should be visable, but shows 00.00 on the frontend unless clicked, and even then it is still two decimal points.

There are currently 7 digits in Ghostfolio as you requested here, but Bitcoin has 8, not more, not less. I hope they can update up to 8.

**Reply by gmag11** [reply]
A possible solution could be configuring the number of significant digits. if it is 6 then 0.00012345678 would display as 0.000123457 and 1234.5678 as 1234.57.

**Comment by Tiagus15** [comment]
Looking forward to having an option in Settings to display up to 8 decimals for crypto.
This would make Activities much more useful, since we could inspect unit prices directly without additional calculation between total spent and total transacted.
Many crypto assets (SHIB, etc.) have values < 0.01, so displaying 0.00 is misleading when the actual values are stored precisely in the DB.
